core/src/main/java/com/graphhopper/routing/Path.java

```diff
@@ -161,7 +165,7 @@ public Path setTime(long time) {
     public Path addTime(long time) {
         this.time += time;
         // ORS-GH MOD START
-        times.add(time);
+        times.add((int) time);
```

Comment:
Are we sure that int has sufficient resolution to store time?

Reply:
Fair point. Please note that these are not timestamps but actual amount of time it takes to travel a given edge (or shortcut). As time is given in ms and `Integer.MAX_VALUE = 2147483647` the longest time period which can be stored corresponds to almost 600h, which seems more than enough to me. The original implementation used `long` rather than `int`, but this requires some additional code to define a corresponding data structure and handle the inversion of the array which I tried to avoid.